spec/lib/listen/event/processor_spec.rb

```diff
@@ -132,15 +132,15 @@ def status_for_time(time)
             it 'still does not process events because it is paused' do
               # pretend we were woken up at 0.6 seconds since start
               allow(config).to receive(:sleep).
-                with(anything) { |*_args| state[:time] += 2.0 }
+                with(anything) { |*_args| state[:monotonic_time] += 2.0 }
```

Comment:
Is it necessary to rename this field?

Reply:
I found it helpful to keep clear that that was not the `Time.now` stub value but rather the `MonotonicTime.now` stub value.
It looked to me like it was just used in this test, so I didn't see a downside to clarifying the name.
Did I get that wrong?

Reply:
I'm not sure I was just concerned it was part of a public interface.

Reply:
My read is that it's strictly internal to this test. The only reason it's in a hash is so it can be mutated as the test runs, to mimic time advancing. (An `@` variable in the test would work that way too.)